ratenyu/courses/views.py

```diff
@@ -11,13 +18,52 @@ def course_detail(request: HttpRequest, course_id: str):
         professors_list = [cl.professor for cl in classes]
         reviews_list = create_review_objects(classes)
         reviews_avg = calculate_rating_avg(reviews_list)
-        context = {
-            "classes": classes,
-            "course": course,
-            "reviews_list": reviews_list,
-            "reviews_avg": reviews_avg,
-            "professors_list": professors_list,
-        }
+        context = {"classes": classes, "course": course, "reviews_list": reviews_list}
```

Comment:
There is an impact of this change on course detail page, list of professor and average review is not visible.

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/19339493/200724688-83ba276a-35a8-4bea-80da-f57d73ce63ff.png">

Reply:
nice catch. just pushed a another commit to fix this